rpp33-boc-atlantic/garager
# Thao feature backend for post

Update:
- Add tests (will fail if we don't have at least 1 user in the database)
- Did not have a method to flush out the data after testing 

To test:
- If there is no user in users table, must add 1 before it can post.

This PR is built on top of PR#58.
PR#58 includes:

- All required field must be filled before move to next step.
- Pricing has to follow 0.00 format.
- Red frame when required field is not filled or filled incorrectly.
- When Name Your Own Price button is check, minimum accepted price is required.
- Zip code is validated.
- Address is not allowed to be type in manually, must be select from drop down
- Able to drag and drop and preview uploaded images.

In this PR I was able to:
- Post to local database

Future plan:

- Tests
- Testing on deployed DB
- Link to listing / stripe set up from confirmation page
